ajax/libs/videojs-background/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-background",
+  "author": "Matthew Harrison-Jones",
+  "filename": "videojs-background.min.js",
+  "description": "Full background video for video.js",
+  "license": "MIT",
+  "keywords": [
+    "videojs",
+    "html5",
+    "flash",
+    "video",
+    "player",
+    "background"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthojo/videojs-Background.git"
+  },
+  "npmName": "videojs-background",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
By expanding these both to the root of the package, we are going to get a clash as files are named the same things in these two dirs:

```
├── dist
│   ├── videojs-background.js
│   └── videojs-background.min.js
├── lib
│   └── videojs-Background.js
```